securedrop/client.py

```diff
@@ -3,10 +3,14 @@
 import os
 import hashlib
 import base64
+import Crypto
```

Comment:
i'm not sure if there's a point in having this import.

Reply:
I'll just import Crypto.Utils.Padding